modules/programs/thunderbird.nix

```diff
@@ -573,13 +595,23 @@ in {
           (attrValues profile.feedAccounts);
 
         accounts = emailAccounts ++ feedAccounts;
+
+        orderedAccountsIds = let
+          accountsOrderIds =
+            map (a: builtins.hashString "sha256" a) profile.accountsOrder;
+          enabledAccountsIds = map (a: a.id) accounts;
+        in accountsOrderIds
```

Comment:
If the intent of this is to allow the ordering of imperative accounts, this is both undocumented and does not work as intended, as imperative accounts do not use the SHA256 hash in their name.

Reply:
If this is fixed it would also be a good idea to not include the `account1` again below when it is already in the list.